components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java

```diff
@@ -917,6 +921,40 @@ public String getPassiveSTSWReply() {
         return passiveSTSWReply;
     }
 
+    /**
+     * Returns the passive sts wreply logout url.
+     *
+     * @return Passive sts wreply logout url.
+     */
+    public String getPassiveSTSWReplyLogout() {
+
+        if (passiveSTSWReplyLogout != null) {
+            return passiveSTSWReplyLogout;
+        }
+
+        InboundAuthenticationRequestConfig[] authRequest = serviceProvider
+                .getInboundAuthenticationConfig().getInboundAuthenticationRequestConfigs();
+
+        if (authRequest != null) {
```

Comment:
Shall we use stringutils in not blank for this? please check other places as well

Reply:
Cannot use StringUtils.isNotBlank() as authRequest object is not a String. Only Strings can be used with StringUtils.isNotBlank().